test/f1.js

```diff
@@ -319,7 +319,7 @@ describe("Basic tests for Zq", () => {
                 f1.putInt(pB, v[j]);
 
                 // eq
-                assert.equal( f1.int_eq(pA,pB), (i==j));
+                assert.equal( f1.int_eq(pA,pB), +(i==j));
```

Comment:
why the +?

Reply:
`fi.int_eq(...)` returns a `number`.  `i==j` returns a boolean.  `+true === 1` and `+false === 0` in JavaScript, so this is just a simple way to typecast a boolean to a number for comparison against another number without implicit typecasting occurring.